src/genesis.rs

```diff
@@ -0,0 +1,94 @@
+use {
+    crate::ValidatorType,
+    log::*,
+    rand::Rng,
+    solana_core::gen_keys::GenKeys,
+    solana_sdk::signature::{write_keypair_file, Keypair},
+    std::{error::Error, path::PathBuf, result::Result},
+};
+
+pub struct Genesis {
+    config_dir: PathBuf,
+    key_generator: GenKeys,
+}
+
+impl Genesis {
+    pub fn new(solana_root: &PathBuf) -> Self {
+        let config_dir = solana_root.join("config-k8s");
+        if config_dir.exists() {
+            std::fs::remove_dir_all(&config_dir).unwrap();
+        }
```

Comment:
nit: maybe could have a subfolder that named by an unique hash 🤔 I guess that I couldn't run two cluster in the same time in the current logic

Reply:
ya so i was going to add that functionality at the end. but it may make sense to add it now. probably easier. do you want to pull a specific git hash from `agave/` or also any forked repo of `agave`?

Reply:
I think we can do both 🤔 (am I too greedy haha)

I think I will have a buildkite pipeline UI for this tool after we done so I will expect that may be two or more people trigger at the same time

Reply:
ok sounds good. its definitely on my list! and in the monorepo work that i am porting here. we can deploy multiple clusters but only through release-channels. commits will be good. it's a little more complicated for deploying multiple validator versions so that's why i was saving until later